home.html

```diff
@@ -35,16 +35,59 @@
           </ul>
         </div>
         <i class="fa fa-bars" onclick="showMenu()"></i>
-        <!-- Building the burger icon below-->
-        <!-- <i class="fa fa-bars"></i> -->
       </nav>
-      <div class="text-box">
+      <!-- Slideshow container -->
+      <div class="slideshow-container">
+        <!-- Full-width images with number and caption text -->
+        <div class="mySlides fade">
+          <div class="numbertext">1 / 3</div>
+          <img src="./style/images/HomePageImage.png" style="width: 100%" />
+          <div class="text">
+            <p>
+              Notre équipe ne ménage aucun effort pour la réussite scolaire et
+              professionnelle de nos chers / chères clients(es).
+            </p>
+          </div>
+        </div>
+
+        <div class="mySlides fade">
+          <div class="numbertext">2 / 3</div>
+          <img src="./style/images/HomePageImage2.png" style="width: 100%" />
+          <div class="text">
+            <p>
+              Un accompagnement éducatif et personnel tout au long du parcours.
+            </p>
+          </div>
+        </div>
+
+        <div class="mySlides fade">
+          <div class="numbertext">3 / 3</div>
+          <img src="./style/images/HomePageImage3.png" style="width: 100%" />
+          <div class="text">
+            <p>
+              Nous travaillons ensemble en étroite collaboration avec les
+              parents et les tuteurs(trices) afin d'offrir une prestation de
+              qualité pour mieux s'orienter.
+            </p>
+          </div>
+        </div>
+
+        <!-- Next and previous buttons -->
+        <a class="prev" onclick="plusSlides(-1)">&#10094;</a>
+        <a class="next" onclick="plusSlides(1)">&#10095;</a>
+      </div>
+      <br />
+
+      <!-- The dots/circles -->
+      <div style="text-align: center">
+        <span class="dot" onclick="currentSlide(1)"></span>
+        <span class="dot" onclick="currentSlide(2)"></span>
+        <span class="dot" onclick="currentSlide(3)"></span>
+      </div>
+      <!-- <div>
```

Comment:
@AmaniLouendriz again, if you're going to comment it out, you may as well remove it.